warehouse/query-core/src/test/java/datawave/query/QueryPlanTest.java

```diff
@@ -112,7 +115,7 @@ public void planInMetricsAfterMissingIndexException() throws Exception {
     @Test
     public void planInMetricsAfterTableNotFoundException() throws Exception {
         String query = Constants.ANY_FIELD + " != " + "'" + TestCities.london + "'";
-        String expectedPlan = "!(_ANYFIELD_ == 'london')";
+        String expectedPlan = "_ANYFIELD_ != 'london'";
```

Comment:
same comment as above, can we run both versions of not equals

Reply:
+1